packages/react/src/css/index.ts

```diff
@@ -36,7 +36,9 @@ export default function css<TProps = unknown>(
   ...interpolations: CssFunction<TProps>[]
 ): CSSProps<TProps>;
 
-export default function css<T = unknown>(styles: CssObject<T> | CssObject<T>[]): CSSProps<T>;
+export default function css<T = unknown>(
+  styles: CssObject<T> | CssObject<T>[] | CSSProps<T> | CSSProps<T>[]
```

Comment:
Will this allow something like this?  Or maybe it looks like it already allows it because `CSSProps<T> extends CssObject<T>` (effectively).
```tsx
const oneStyles = css({ color: 'red' });
const twoStyles = css(oneStyles);
```

Reply:
Yeah 😬.